templates/alertmanager.go

```diff
@@ -0,0 +1,14 @@
+package templates
+
+import (
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/red-hat-storage/ocs-operator/v4/controllers/defaults"
+	"k8s.io/utils/ptr"
+)
+
+var AlertmanagerTemplate = promv1.Alertmanager{
+	Spec: promv1.AlertmanagerSpec{
+		Replicas:  ptr.To(int32(1)),
+		Resources: defaults.MonitoringResources["alertmanager"],
+	},
+}
```

Comment:
If the only thing we need is the spec, why don't we have a spec template?
```suggestion
var AlertmanagerSpecTemplate = promv1.AlertmanagerSpec{
	Replicas:  ptr.To(int32(1)),
	Resources: defaults.MonitoringResources["alertmanager"],
}
```

Reply:
Updated